client/src/components/categorical/annoDialog.js

```diff
@@ -18,6 +18,8 @@ class AnnoDialog extends React.PureComponent {
       validationError,
       annoSelect,
       annoInput,
+      secondaryInstructions,
```

Comment:
This component is now used by `.../components/categorical/` and `.../components/geneExpression/`.  Seems like it nees to be moved up the tree into `.../components`, so geneExpression has no dependency on internal categorical sub-components.